data_adapter_oemof/build_datapackage.py

```diff
@@ -63,6 +63,28 @@ def refactor_timeseries(timeseries: pd.DataFrame):
 
     return df_timeseries
 
+def add_bus_to_element_dict(elements_dict):
+    """
+    Takes a sequence dict of processes with sequence data and searches for busses in columns that contain `bus`
+    :param elements_dict:
+    :return: Updated Sequence with additional entry "bus" that contains all occurring bus entries as balanced busses.
+    """
+    unique_entries = set()
+    for element in elements_dict.values():
+        bus_occourences = [col for col in element.columns if "bus" in col]
```

Comment:
Do we have to use a different approach to find all columns containing the busses?